orcid-integration-test/src/test/java/org/orcid/integration/blackbox/api/v2/rc2/PersonTest.java

```diff
@@ -120,6 +121,18 @@ public void testViewPersonFromMemberAPI() throws InterruptedException, JSONExcep
         assertNotNull(accessToken);
         ClientResponse response = memberV2ApiClient.viewPerson(this.user1OrcidId, accessToken);
         assertNotNull(response);
+        //Occasionaly gives 500 when run as Suite.  Passes on it's own...?
+        /*
+            <error xmlns="http://www.orcid.org/ns/error">
+                <response-code>500</response-code>
+                <developer-message>org.hibernate.LazyInitializationException: failed to lazily initialize a collection of role: org.orcid.persistence.jpa.entities.ProfileEntity.givenPermissionTo, could not initialize proxy - no Session</developer-message>
+                <user-message>Something went wrong in ORCID.</user-message>
+                <error-code>9008</error-code>
+                <more-info>http://members.orcid.org/api/api-error-codes</more-info>
+            </error>
+         */
+        assertEquals("invalid "+response,200,response.getStatus());
```

Comment:
@TomDemeranville I just see the same issue: https://trello.com/c/ubXWd5TC/2619-500-error